src/modules/layerManager/components/LayerItem.js

```diff
@@ -285,6 +285,7 @@ export class LayerItem extends AbstractMvuContentPanel {
 						.icon="${oafSettingsSvg}"
 						.title=${translate('layerManager_oaf_settings')}
 						.color=${'var(--secondary-color)'}
+						.size=${1.8}
```

Comment:
![grafik](https://github.com/user-attachments/assets/0c8f2952-f670-4bd2-b920-b5d22c664284)

@alsturm Let's adjust the size of the icons?

Reply:
@alsturm  Also let's use a `gear` icon for the settings